scripts/js/queries.js

```diff
@@ -245,30 +245,41 @@ function formatReplyTime(replyTime, type) {
   return replyTime;
 }
 
-function formatInfo(data) {
-  // DNSSEC status
-  var dnssecStatus = data.dnssec,
-    dnssecClass;
-  switch (data.dnssec) {
+// Parse DNSSEC status
+function parseDNSSEC(data) {
+  var icon = null, // Icon to display
+    color = false, // Class to apply to text
+    text = data.dnssec; // Text to display
+  switch (text) {
     case "SECURE":
-      dnssecClass = "text-green";
+      icon = "fa-solid fa-lock";
+      color = "text-green";
       break;
     case "INSECURE":
-      dnssecClass = "text-orange";
+      icon = "fa-solid fa-unlock";
```

Comment:
The icons `fa-lock` and `fa-unlock` are almost identical, specially in small sizes. 
It is hard to distinguish them.
*(There is just a small gap representing the lock is unlocked)*. 

```suggestion
      icon = "fa-solid fa-lock-open";
```

We show use `fa-lock-open`.